docs/webapp/pipelines/webapp_pipeline_page.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Pipelines Page
+---
+
+Use the **Pipelines** Page to navigate between and manage pipelines. The page shows execution summaries for all 
+[ClearML Pipelines](../../fundamentals/pipelines.md).
+
+Click on a pipeline card to navigate to its [Pipeline Runs Table](webapp_pipeline_table.md), where you can view the 
+pipeline structure, configuration and outputs for all of the pipeline’s runs, as well as create new runs.
+
+![Projects page](../../img/webapp_pipeline_table.png)
```

Comment:
@pollfly Isn't this (and following images) "Pipelines" (as opposed to "Projects") image labels?